app/src/renderer/vuex/modules/keybase.js

```diff
@@ -75,3 +69,35 @@ export default () => {
     mutations
   }
 }
+
+async function lookupId(state, keybaseId) {
+  const urlPrefix = `https://keybase.io/_/api/1.0/user/lookup.json?key_suffix=`
+  const fullUrl = urlPrefix + keybaseId
+  const json = await state.externals.axios(fullUrl)
+  if (json.data.status.name === `OK`) {
+    const user = json.data.them[0]
+    if (user && user.pictures && user.pictures.primary) {
+      return {
+        avatarUrl: user.pictures.primary.url,
+        userName: user.basics.username,
+        profileUrl: `https://keybase.io/` + user.basics.username
+      }
+    }
+  }
+}
+
+async function lookupUsername(state, username) {
+  const urlPrefix = `https://keybase.io/_/api/1.0/user/lookup.json?usernames=`
```

Comment:
i believe you're still requesting the full user object...

```
User objects are massive

...

To avoid getting more data than you need, you can (should!) specify a comma-separated list of fields you desire. Acceptable values are:

pictures
basics
public_keys
profile
proofs_summary
remote_key_proofs
sigs
cryptocurrency_addresses NEW
```

from https://keybase.io/docs/api/1.0/call/user/lookup

Reply:
Nice tip